### PR TITLE
fix 404 urls number six

### PR DESCRIPTION
**Pull request by CriaHu**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
>
> For pull requests on Chinese localization, set Base Branch to release-1.16
> Feel free to ask questions in #kubernetes-docs-zh
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by CriaHu** [comment]
#18228  fix number six

**Comment by onlydole** [comment]
Thank you so much for this, @CriaHu

/lgtm
/assign @kbarnard10

**Comment by tanjunchen** [comment]
ref:https://github.com/kubernetes/website/issues/17649

File: content/en/blog/_posts/2016-04-00-Building-Awesome-User-Interfaces-For-Kubernetes.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ SIG UI members have been independently working on a variety of UIs for Kubernete
...
+Since the initial release of the Dashboard UI we have been thinking hard about what to do next and what users of UIs for Kubernetes think about our plans. We’ve had many internal discussions on this topic, but most importantly, reached out directly to our users. We created a questionnaire asking a few demographic questions as well as questions for prioritizing use cases. We received more than 200 responses from a wide spectrum of user types, which in turn helped to shape the Dashboard UI’s [current roadmap](https://github.com/kubernetes/dashboard/blob/master/docs/common/roadmap.md). Our members from LiveWyer summarised the results in a [nice infographic](http://static.lwy.io/img/kubernetes_dashboard_infographic.png).&nbsp;
```
Comment (sftim) [inline content/en/blog/_posts/2016-04-00-Building-Awesome-User-Interfaces-For-Kubernetes.md]: Looks like the roadmap is actually at https://github.com/kubernetes/dashboard/wiki/Roadmap

_but_ it's an old blog post, it's OK for blog posts to have broken old links

(It's _even cooler_ if the target URIs don't change).

Maybe it's better to link to https://github.com/kubernetes/dashboard/blob/4932fa7b50619849d2d68e55342925938e42e881/docs/devel/roadmap.md ? That's where the page pointed when the blog was written.

**Comment by tanjunchen** [comment]
@CriaHu 
update?

**Comment by kbarnard10** [comment]
@CriaHu I'm going to close this for now since it's been hanging out here awhile. Please feel free to reopen when you have time to address the feedback. Thank you!

**Pull request closed without merging**